lib/ansible/plugins/inventory/linode.py

```diff
@@ -193,6 +181,9 @@ def verify_file(self, path):
 
     def parse(self, inventory, loader, path, cache=True):
         """Dynamically parse Linode the cloud inventory."""
+        if not HAS_LINODE_API4:
+            raise AnsibleParserError('the Linode dynamic inventory plugin requires linode_api4.')
```

Comment:
Oh why not :sweat_smile: 

```suggestion
            raise AnsibleParserError('The Linode dynamic inventory plugin requires linode_api4.')
```

Reply:
Heh, just a copy-pasta from what it was before :-)